get_file.go

```diff
@@ -103,6 +108,11 @@ func (g *FileGetter) GetFile(ctx context.Context, req *Request) error {
 		return err
 	}
 
+	if req.Inplace {
+		req.Dst = path
+		return nil
+	}
+
```

Comment:
Should this be moved above `MkdirAll` call or Dst presence checks?  I'd assume so if we wanted to avoid any operation when running in place.

Reply:
Yes this is totally correct ! It's been a while but I wonder why it did this. Moving this up before the Lstat of the dst